src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -481,16 +473,10 @@ public Object onTraversalSuccess() {
     protected void onShutdown() {
         super.onShutdown();
 
-        if ( engine != null ) {
+        if (engine != null) {
             logger.info(String.format("***Processed %d total sites", engine.getTotalNumberOfSites()));
             logger.info(String.format("***Processed %d total variants", engine.getTotalNumberOfVariants()));
             logger.info(String.format("***Read API scanned %d bytes", engine.getTotalEstimatedBytesScanned()));
         }
-
-        // Close up our writer if we have to:
-        if ( vcfWriter != null ) {
-            vcfWriter.close();
-        }
```

Comment:
why dont we need to close this any more?

Reply:
It's closed in the VCF writer subclass that actually does the writing now.